.gitignore

```diff
@@ -2,3 +2,4 @@ node_modules
 coverage
 *.log
 .rpt2_cache
+test/dist
```

Comment:
I think we can commit the `test/dist` as well. It is not going to change often is it? What do you think?

Reply:
This isn't needed - test/dist is generated everytime before test and it's just required for travis ci or local tests.